### ISSUE TITLE
"Failed to load resource, kCFErrorDomainCFNetwork, code 1", OSX, Safari 9

### ISSUE THREAD

**Issue by fedorinoGore** [futurepress/epub.js#311]
While trying to load a book with ionic web-simulation I get two errors:
first one:
"Could not load book at: META-INF/container.xml"
  (анонимная функция)book.js:203
  lib$rsvp$$internal$$tryCatchrsvp.js:492
  lib$rsvp$$internal$$invokeCallbackrsvp.js:504
  lib$rsvp$$internal$$publishrsvp.js:475
  lib$rsvp$$internal$$publishRejectionrsvp.js:418
  lib$rsvp$asap$$flushrsvp.js:1197
and another one:file:///Users/pro/Development/ionic/.../moby-dick/META-INF/container.xml
 "The operation couldn’t be completed. (kCFErrorDomainCFNetwork, code 1)".

This error persist only in browser, in iOS9.1 simulator all seems fine.
(Other errors popping out, but it is a matter of other issue)

**Comment by fchasen** [comment]
Look like the book can't be loaded. Are you able to access the `META-INF/container.xml` file directly?

**Comment by fedorinoGore** [comment]
Yes, it is located in correct directory (mentioned in error message) and I could open it with text editor of my choice.

**Comment by fchasen** [comment]
Can you access it in the browser though? I'm guessing the filesystem won't server .xml files.
